DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -184,6 +184,19 @@ internal async Task<IReadOnlyList<DiscordMember>> SearchMembersAsync(ulong guild
             return mbrs;
         }
 
+        internal async Task<DiscordBan> GetGuildBanAsync(ulong guild_id, ulong user_id)
+        {
+            var route = $"{Endpoints.GUILDS}/:guild_id{Endpoints.BANS}/:user_id";
+            var bucket = this.Rest.GetBucket(RestRequestMethod.GET, route, new {guild_id, user_id}, out var url);
+            var uri = Utilities.GetApiUriFor(url);
+            var res = await this.DoRequestAsync(this.Discord, bucket, uri, RestRequestMethod.GET, route).ConfigureAwait(false);
+            var json = JObject.Parse(res.Response);
+
+            var ban = json.ToDiscordObject<DiscordBan>();
```

Comment:
This should be `JsonConvert.DeserializeObject<DiscordBan>(res.Response)` (for consistency)

Reply:
I've been implored to use .ToDiscordObject > ToObject, but I'll save that overhaul for another PR